### PR TITLE
app management page

### PR DESCRIPTION
**Pull request by MonicaG**
This page describes the permission levels users have in App Store Connect and Google Play Console.

It uses updated info from this [page on the wiki](https://github.com/bcgov/mobile-signing-service/wiki/Google-App-Signing#developer-experience-teams-responsibility) and [this section](https://developer.gov.bc.ca/Mobile-Starter-Kit-Overview#management) on the developer kit

### CODE REVIEW

File: docs/app_management.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* Developer the app
+* Respond to user reviews
+
+We will use this information to setup your team members on the app stores.  
+
+## Managing User Access To App Store Connect
+
+[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. Roles available to app team members are: 
+
+* App Manager
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:15]:
```suggestion

```

File: docs/app_management.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* Customer Support
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:16]:
```suggestion

```

File: docs/app_management.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* Developer
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:17]:
```suggestion

```

File: docs/app_management.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* Marketing
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:18]:
```suggestion

```

File: docs/app_management.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+* Sales
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:19]:
```suggestion

```

File: docs/app_management.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. Roles available to app team members are: 
+
+* App Manager
+* Customer Support
+* Developer
+* Marketing
+* Sales 
+
+A person may have more than one role. For example, a user may have both the App Manager and Customer Support roles. Please consult Apple's [Program Roles](https://developer.apple.com/support/roles/) documentation for the functions available to each role. The following sections describe the purpose of each role.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:21]:
```suggestion

```

File: docs/app_management.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+### Sales Role
+Users in the Sales role can view app analytics, sales and trends reports. This role is usually assigned to the Product Owner. But, some teams have dedicated team members in this role.
+
+
+
+
+### Apple ID
+Any email address may be used to create an account on App Store Connect. It does not need to be one associated with an Apple ID. But, the user will [need to create an Apple ID](https://developer.apple.com/help/app-store-connect/manage-your-team/add-and-edit-users) when activating the account.
+
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:44]:
```suggestion
A person may have more than one role. For example, a user may have both the App Manager and Customer Support roles. Please consult Apple's [Program Roles](https://developer.apple.com/support/roles/) documentation for the functions available to each role. The following sections describe the purpose of each role.
```

File: docs/app_management.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+## Managing User Access To Google Play Console
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:46]:
```suggestion
## Managing user access to Google Play Console
```

File: docs/app_management.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The developer experience team will create three permission groups for your app:
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:48]:
```suggestion
[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The Developer Experience Team will create 3 permission groups for your app:
```

File: docs/app_management.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+* Admin
+* Release
+* Report
+
+Users will then be assigned into one, or more, or these permission groups.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:54]:
```suggestion
Users will then be assigned into one, or more, of these permission groups. 
```

File: docs/app_management.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The developer experience team will create three permission groups for your app:
+
+* Admin
+* Release
+* Report
+
+Users will then be assigned into one, or more, or these permission groups. 
+
+Please see the "Permission definitions and uses" section in Google's [Add developer account users and manage permissions]( https://support.google.com/googleplay/android-developer/answer/9844686) documentation. It provides a complete list of the permissions available.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:56]:
```suggestion
See the "Permission definitions and uses" section in Google's [Add developer account users and manage permissions]( https://support.google.com/googleplay/android-developer/answer/9844686) documentation. It provides a complete list of the permissions available.
```

File: docs/app_management.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### Admin Permission Group
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:59]:
```suggestion
### Admin permission group
```

File: docs/app_management.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+Users in this group have full access to the app. Their permissions also include inviting new users to the developer account. They can also add and remove permissions for other users who have access to the same apps. The Product and Technical Owners are members of this group.
+
+### Release Permission Group
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:62]:
```suggestion
### Release permission group
```

File: docs/app_management.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+The permissions available to this group include the following:
+* Release to production, exclude devices and use Play app signing
+* Release apps to testing tracks
+* Manage testing tracks and edit tester lists
+
+The developer experience team will work you to decide which options you would like to enable.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:68]:
```suggestion
The Developer Experience Team will work with you to decide which options you'd like to enable. 
```

File: docs/app_management.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+### Release Permission Group
+The permissions available to this group include the following:
+* Release to production, exclude devices and use Play app signing
+* Release apps to testing tracks
+* Manage testing tracks and edit tester lists
+
+The developer experience team will work you to decide which options you would like to enable. 
+
+Users in the [Admin Permission Group](#admin-permission-group) can administer this group. This allows them to add or remove users from the group. The Admin Group can also enable or disable the permission options listed above.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:70]:
```suggestion
Users in the [admin permission group](#admin-permission-group) can administer this group. This allows them to add or remove users from the group and enable or disable permission options.
```

File: docs/app_management.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### Report Permission Group
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:73]:
```suggestion
### Report permission group
```

File: docs/app_management.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+This permission group lets a user view all information for this app, but not financial data. The permission to view Financial Data can be enabled at the discretion of the Product Owner. Users in the [Admin Permission Group](#admin-permission-group) for the app can administer this group.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:74]:
```suggestion
This permission group lets a user view all information for this app, but not financial data. The permission to view financial data can be enabled at the discretion of the Product Owner. Users in the [admin permission group](#admin-permission-group) for the app can administer this group.
```

File: docs/app_management.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### Google Account
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:77]:
```suggestion
### Google account
```

File: docs/app_management.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+A user does not need a Gmail account in order to make a Google account. Please refer to the "Use an existing email address" section in Google's [Create a Google Account](https://support.google.com/accounts/answer/27441?hl=en) documentation.
```
Comment (CoffeeGurlNia) [inline docs/app_management.md:78]:
```suggestion
A user doesn't need a Gmail account in order to make a Google account. Refer to the "Use an existing email address" section in Google's [Create a Google Account](https://support.google.com/accounts/answer/27441?hl=en) documentation.
```

File: docs/app_management.md [hunk 20]
```diff
@@ -10,9 +10,11 @@ We'll use this information to setup your team members on the app stores.
 
 ## Managing user access to App Store Connect
 
-[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. Roles available to app team members are: 
+[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. 
 
+A person may have more than one role. For example, a user may have both the App Manager and Customer Support roles. Please consult Apple's [Program Roles](https://developer.apple.com/support/roles/) documentation for the functions available to each role. 
 
+Roles available to app team members are:
```
Comment (MonicaG) [inline docs/app_management.md:18]: @CoffeeGurlNia - I reorganized the section a bit. I've moved the section about multiple roles to above the list of roles. I think it should be separate from the Apple ID section

File: docs/app_management.md [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+
+
+
+### Apple ID
+Any email address may be used to create an account on App Store Connect. It does not need to be one associated with an Apple ID. But, the user will [need to create an Apple ID](https://developer.apple.com/help/app-store-connect/manage-your-team/add-and-edit-users) when activating the account.
+
+
+## Managing user access to Google Play Console
+
+[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The Developer Experience Team will create 3 permission groups for your app:
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:44]:
```suggestion
[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The Developer Experience team will create 3 permission groups for your app:
```

File: docs/app_management.md [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+
+* Admin
+* Release
+* Report
+
+Users will then be assigned into one, or more, of these permission groups.
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:50]:
```suggestion
Users are then assigned to one or more permission groups. 
```

File: docs/app_management.md [hunk 23]
```diff
@@ -0,0 +1,74 @@
...
+
+[Play Console](https://accounts.google.com/ServiceLogin?service=androiddeveloper&passive=true&continue=https%3A%2F%2Fplay.google.com%2Fconsole%2Fdeveloper%2F&_ga=2.124271306.1978797360.1688067469-933612381.1687381212) is the platform that allows teams to manage their apps on the Google Play Store. It uses permission settings to control who has access to do certain functions. The Developer Experience Team will create 3 permission groups for your app:
+
+* Admin
+* Release
+* Report
+
+Users will then be assigned into one, or more, of these permission groups. 
+
+See the "Permission definitions and uses" section in Google's [Add developer account users and manage permissions]( https://support.google.com/googleplay/android-developer/answer/9844686) documentation. It provides a complete list of the permissions available.
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:52]:
```suggestion
Review the "Permission definitions and uses" section in Google's [Add developer account users and manage permissions]( https://support.google.com/googleplay/android-developer/answer/9844686) documentation. It provides a complete list of the permissions available.
```

Comment (kaitlyn-rosenburg) [reply docs/app_management.md:52]: don't use "see" as it implies the reader is a sighted person

File: docs/app_management.md [hunk 24]
```diff
@@ -0,0 +1,74 @@
...
+
+
+### Admin permission group
+Users in this group have full access to the app. Their permissions also include inviting new users to the developer account. They can also add and remove permissions for other users who have access to the same apps. The Product and Technical Owners are members of this group.
+
+### Release permission group
+The permissions available to this group include the following:
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:59]:
```suggestion
The permissions available to this group include:
```

File: docs/app_management.md [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+* Release to production, exclude devices and use Play app signing
+* Release apps to testing tracks
+* Manage testing tracks and edit tester lists
+
+The Developer Experience Team will work with you to decide which options you'd like to enable.
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:64]:
```suggestion
The Developer Experience team will work with you to decide which options you'd like to enable. 
```

File: docs/app_management.md [hunk 26]
```diff
@@ -0,0 +1,74 @@
...
+* Release apps to testing tracks
+* Manage testing tracks and edit tester lists
+
+The Developer Experience Team will work with you to decide which options you'd like to enable. 
+
+Users in the [admin permission group](#admin-permission-group) can administer this group. This allows them to add or remove users from the group and enable or disable permission options.
+
+
+### Report permission group
+This permission group lets a user view all information for this app, but not financial data. The permission to view financial data can be enabled at the discretion of the Product Owner. Users in the [admin permission group](#admin-permission-group) for the app can administer this group.
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:70]:
```suggestion
This permission group lets a user view all information for this app (except financial data). The permission to view financial data can be enabled at the discretion of the Product Owner. Users in the [admin permission group](#admin-permission-group) for the app can administer this group.
```

File: docs/app_management.md [hunk 27]
```diff
@@ -0,0 +1,74 @@
...
+
+
+### Google account
+A user doesn't need a Gmail account in order to make a Google account. Refer to the "Use an existing email address" section in Google's [Create a Google Account](https://support.google.com/accounts/answer/27441?hl=en) documentation.
```
Comment (kaitlyn-rosenburg) [inline docs/app_management.md:74]:
```suggestion
A user doesn't need a Gmail account in order to make a Google account. Review to the "Use an existing email address" section in Google's [Create a Google Account](https://support.google.com/accounts/answer/27441?hl=en) documentation.
```

**Review by kaitlyn-rosenburg — CHANGES_REQUESTED** [review]
finished my edit!

**Comment by MonicaG** [comment]
@kaitlyn-rosenburg - Thank you for your review! I've committed the changes you suggested. You can [review them here](https://github.com/bcgov/mobile-developer-guide/pull/12/commits/8244bb853066663aeb827bd2f1589e75de2b9c4a).  I've also updated ['Role' to 'role' for all of the headings](https://github.com/bcgov/mobile-developer-guide/pull/12/commits/9448048e640b45691ab149961309063766604581).

**Review by kaitlyn-rosenburg — APPROVED** [review]

**Pull request merged**